src/resource.ts

```diff
@@ -597,6 +600,13 @@ export class BaseResource implements IResource {
         const fieldType = resourceType.fields.find(
           (field: GQLType) => field.name === inputName,
         )
+
+        // if (inputName == 'coords' && fieldType) {
+        //   const foo = this.recordToVariables(input[inputName], fieldType.type)
```

Comment:
fieldType.type is no inputtype and has no ` inputFields` so it can't be use for recordToVariables :-(